single-pool/js/packages/client/src/index.ts

```diff
@@ -0,0 +1,21 @@
+import { getBase58EncodedAddressCodec } from '@solana/web3.js';
+
+import { PoolAddress, VoteAccountAddress } from './addresses';
+
+export * from './addresses';
+export * from './instructions';
+export * from './transactions';
+
+export async function getVoteAccountAddressForPool(
+  rpc: any, // XXX not exported: Rpc<GetAccountInfoApi>,
+  poolAddress: PoolAddress,
+  // XXX not exported: abortSignal?: AbortSignal
```

Comment:
Add `DOM` to the `lib` in your `tsconfig.json`. It's a DOM builtin, and [available in Node >14.17.0](https://nodejs.org/dist/latest-v18.x/docs/api/globals.html#class-abortsignal).